pyogrio/tests/test_geopandas_io.py

```diff
@@ -196,61 +194,82 @@ def test_read_no_geometry_no_columns_no_fids(naturalearth_lowres, use_arrow):
         )
 
 
-def test_read_force_2d(test_fgdb_vsi, use_arrow):
-    with pytest.warns(
```

Comment:
Verifying the warning was incidental to this test, and already covered by `test_core.py::test_list_layers`.